lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_kafka.conf

```
@@ -62,7 +62,7 @@ emqx_ee_bridge_kafka {
     mqtt_topic {
         desc {
             en: "MQTT topic or topic as data source (bridge input).  Should not configure this if the bridge is used as a rule action."
-            zh: "指定 MQTT 主题作为桥接的数据源。 若该桥接用于规则的动作，则必须将该配置项删除。"
+            zh: "桥接指定 MQTT 主题数据，也可以留空由规则动作指定数据源。"
```

@lina50 I think we should use a more assertive tune here.
In practice, no one should need to configure this if the bridge is used as a rule action.

```suggestion
            zh: "指定源 MQTT 主题，若由规则动作指定数据源，则该配置应留空，否则消息会被重复发送到 Kafka。"
```